css/main.css

```diff
@@ -111,14 +246,28 @@ body {
 .section_1 {
   background-color: #fff;
 }
+.head_section1 {
+  width: 445px;
+  height: 87px;
+  font-size: 36px;
+  line-height: 43px;
+}
+.block_for_section_1 {
+  margin-top: 140px;
+  margin-bottom: 102px;
+}
 .text_section_1 {
+  width: 445px;
+  height: 115px;
   font-size: 16px;
-  line-height: 22px;
+  line-height: 140%;
 }
 .video {
   width: 540px;
   height: 321px;
   border-radius: 16px;
+  margin-top: 94px;
+  margin-bottom: 94px;
```

Comment:
It should be paddings for section, not margins for video.

Reply:
I've deleted these margins and added a new class with "paddings" to div element which covered my video.
.video-wrapper {
  padding-top: 94px;
  padding-bottom: 94px;
}